definitions.json

```diff
@@ -93,15 +81,15 @@
                     "issued_tokens": "Wrapped",
                     "to_be_redeemed_tokens": "Wrapped",
                     "wallet": "Wallet",
-                    "backing_collateral": "Collateral",
```

Comment:
What will be the recommended way of reading the vault's collateral (to know which are active)? Is it via the Collateral pallet?

Reply:
We'll still have the `backing_collateral` as defined below.